src/win/fs.c

```diff
@@ -531,7 +531,7 @@ void fs__close(uv_fs_t* req) {
   if (fd > 2)
     result = _close(fd);
   else
-    result = 0;
+    result = UV__EINVAL;
```

Comment:
This should set `_doserrno` to EINVAL (not `UV__EINVAL` - I think) and result to -1.  The SET_REQ_RESULT macro obscures it but otherwise `req->sys_errno_` won't be initialized.